packages/node_modules/@node-red/runtime/locales/en-US/runtime.json

```
@@ -155,6 +155,7 @@
             "projects": {
                 "changing-project": "Setting active project : __project__",
                 "active-project": "Active project : __project__",
+                "projects-folder": "Projects folder: __projectsFolder__",
```

We use `directory` rather than `folder` when logging the location of the user directory, so should be consistent here:

```suggestion
                "projects-folder": "Projects directory: __projectsFolder__",
```